lib/recording/RecordingManager.js

```diff
@@ -345,10 +345,18 @@ class RecordingManager {
                                 logger.error(`[${entryId}-${recordingEntrySession.recordedEntryId}][stopRecording]: recorded entry is unknown, skip creating soft link`);
                                 return Q.resolve()
                             }
-                            if (recordingEntrySession.serverType !== kalturaTypes.KalturaEntryServerNodeType.LIVE_PRIMARY) {
-                                logger.info(`[${entryId}-${recordingEntrySession.recordedEntryId}][stopRecording] entry is backup, skip creating soft link`);
-                                return Q.resolve()
-                            }
+                        })
+                        .then(() => {
+                            let dcPath = path.join(recordingEntrySession.recordingSessionPath, 'dc');
```

Comment:
i suggest instead of saving "dc" file to save json file with all sorts of metadata (for now it's only dc)
but maybe in the future we will need more stuff)